### DISCUSSION TITLE
YAML

### DISCUSSION THREAD

**Discussion by mportesdev** [underground-lab/eu2020#28] [Show and tell]
Jen pro zajímavost: zkusil jsem, o kolik kompaktnější/čitelnější by data byla ve formátu YAML.

Můžeš se na to podívat, ve větvi `experimental/yaml` je soubor [ev_members_cz.yml](https://github.com/underground-lab/eu2020/blob/experimental/yaml/eu2020/data/ev_members_cz.yml).

Michal

**Comment by mportesdev** [comment]
Syntaxe `satisfaction: { CZ: -3, HU: -3, PL: -3, SK: -1 }` je taky podporovaná, takže nemusí každá hodnota zabírat jeden řádek.

**Reply by berk76** [reply]
Koukám na to a přemýšlím o kolik je to čitelnější. 

Když přelétnu očima ten `dict` tak hned rozeznám 3 události. Když koukám na ten `yml` tak musím hodně ostřit.

Možná mi to přijde nepřehledné protože tam je odsazení jen 2 znaky.

**Comment by mportesdev** [comment]
Na ukázku jsem to ještě trochu přeformátoval:
- odsazení 4
- dva prázdné řádky mezi událostmi
- jednořádkové `satisfaction`

Zatím bych to ale nechal u ledu.

**Reply by berk76** [reply]
Teď už to vypadá mnohem lépe. Časem bychom ty slovníky ale měli stejně předělat buď to `json` nebo do `yml`. Zatím nemám žádnou jasnou preferenci :-)
